apps/spruce/src/pages/task/taskTabs/TaskHistory/CommitDetailsCard/CommitDetailsCard.stories.tsx

```diff
@@ -35,26 +35,44 @@ export const Default: CustomStoryObj<TemplateProps> = {
   render: (args) => <Template {...args} />,
 };
 
+export const WithUnmatchingSearchResult: CustomStoryObj<TemplateProps> = {
+  render: (args) => <RenderWithUnmatchingSearchResult {...args} />,
+};
```

Comment:
nit. you don't have to make a separate story probably, you could expose `isMatching` as a boolean toggle instead (similar to `canRestart`)